app/components/Form/EditorField.js

```diff
@@ -10,31 +16,47 @@ type Props = {
   className?: string,
   input: any,
   meta: any,
-  name: string
+  name: string,
+  uploadFile: (file: Blob) => Promise<*>
 };
 
-class NoSSRError {
-  error: Error;
-  constructor(msg) {
-    this.error = new Error(msg);
-  }
-}
+const mapDispatchToProps = dispatch => {
+  return {
+    uploadFile: async file => {
+      const response = await dispatch(uploadFile({ file, isPublic: true }));
+      return { fileKey: response.meta.fileKey };
+    }
+  };
+};
 
-function EditorField({ className, name, ...props }: Props) {
-  if (!__CLIENT__) {
-    throw new NoSSRError('Cannot SSR editor');
```

Comment:
Are you sure this is ok to remove?

Reply:
I put it back just in case now. You may be right in that it will throw errors still...